indigo/distribution.yaml

```diff
@@ -8583,6 +8583,15 @@ repositories:
       url: https://github.com/xaxxontech/oculusprime_ros.git
       version: master
     status: developed
+  odometry_serialcom:
+    doc:
+      type: git
+      url: https://github.com/wlfws/odometry_serialcom.git
+      version: indigo-devel
```

Comment:
the `version` here should be a valid branch name. Looking at your repository there is a single branch named `master`. you likely want to use `master` instead of `indigo-devel` here (Same comment below for the source entry)